airbyte_cdk/cli/airbyte_cdk/_connector.py

```diff
@@ -162,6 +162,13 @@ def test(
         pytest_args.append("--collect-only")
 
     pytest_args.append(str(test_file_path))
+
+    if connector_directory:
+        build_dir = connector_directory / "build"
```

Comment:
I'll go ahead and apply a commit with this suggestion:

```suggestion
        build_dir = connector_directory / "build" / "test-results"
```

I compared to our other tests in the main repo `poe` file.

`--junitxml=build/test-results/pytest-unit-tests-junit.xml`